src/internal/digitalAsset.ts

```diff
@@ -196,6 +208,27 @@ export interface CreateCollectionOptions {
   royaltyDenominator?: number;
 }
 
+const createCollectionAbi: EntryFunctionABI = {
```

Comment:
do we need to keep an eye on the move contract if any changes?

Reply:
These functions, along with all other entry functions we provide, are public entry, and shouldn't have future conflicts.